admin_app/src/app/integrations/components/ChatManagerCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { Card, CardContent } from "@mui/material";
```

Comment:
Thank you for creating components here!
What are your thoughts on moving @/components/APIKeyModals here as well?

Reply:
Good idea!